apps/web/src/pages/templates/workflow/SideBar/StepSettings.tsx

```diff
@@ -79,18 +149,6 @@ export function StepSettings({ index }: { index: number }) {
         </Button>
       </Group>
       <ReplyCallback index={index} control={control} />
-      <FilterModal
-        isOpen={filterOpen}
-        cancel={() => {
-          setFilterOpen(false);
-        }}
-        confirm={() => {
-          setFilterOpen(false);
-        }}
-        control={control}
-        stepIndex={index}
-        setValue={setValue}
-      />
```

Comment:
maybe you have z-index problems because you are not rendering the conditions in this place?